bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -0,0 +1,208 @@
+#!/bin/env python
+""" Apply a naive mass binning, assuming that each bin is fully independent, which 
```

Comment:
Will want to fix this description eventually

Reply:
Fixed